src/metabase/lib/metadata/cached_provider.cljc

```diff
@@ -50,19 +50,28 @@
 
 (defn- metadatas [cache uncached-provider metadata-type ids]
   (when (seq ids)
-    (log/debugf "Getting %s metadata with IDs %s" metadata-type (pr-str (sort ids)))
```

Comment:
There were conflicts here with changes in the following two PRs, which I resolved in favor of the code in master, i.e. by also backporting the changes from these PRs. The changes seemed safe to backport in isolation, but pinging @piranha  / @alexander-yakushev in case we don't want to backport them (or don't want to partially-backport in the case of alexander's PR).

* #44977
* #46103

Reply:
It's fine to backport mine; thanks for pinging!

Reply:
oh mine's fine too

Reply:
👍